ckanext/xloader/templates/xloader/resource_data.html

```diff
@@ -5,17 +5,35 @@
 {% block primary_content_inner %}
 
   {% set action = h.url_for('xloader.resource_data', id=pkg.name, resource_id=res.id) %}
+  {% set delete_action = h.url_for('xloader.delete_datastore_table', id=pkg.id, resource_id=res.id) %}
```

Comment:
Testing at dev.data.qld.gov.au indicates that setting a variable at this point in the template is ineffective. It needs to be set inside the block where it's used.

The previous line inadvertently gets away with it because a blank action means "submit to the current URL", which is the desired behaviour.

Reply:
@ThrawnCA cool, did not know about variables and blocks scopes.

This should be fixed now.